contrib/simple-client.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: LGPL-2.1+
+
+"""A simple fwupd frontend"""
+import sys
+import os
+import gi
+gi.require_version('Fwupd', '2.0')
+from gi.repository import Fwupd
+from gi.repository import GLib
+
+class Progress ():
+    def __init__ (self):
+        self.device = None
+        self.status = None
+        self.percent = 0
+        self.erase = 0
+
+    def device_changed (self, new_device):
+        if self.device != new_device:
+            self.device = new_device
+            print ("\nUpdating %s" % self.device)
+
+    def status_changed (self, percent, status):
+        if (self.status != status) or (self.percent != percent):
```

Comment:
Does pylint like the brackets?

Reply:
Pylint had a bazillion nits, but not that.  I've adjusted for every nit in pylint and removed the brackets too.